src/main/resources/solr/schemas/acl-anthology.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
Don't think we should include Solr internal fields like `__root__` etc since those are probably not going to change?

Reply:
I agree. What are the internal Solr internal fields? Is it everything with underscores? I'll create a collection with no configuration. Let's see what fields exist

Reply:
Yeah it's just the underlined fields, everything else is from the fields that are actually part of the indexed documents